packages/components/src/input-control/types.ts

```diff
@@ -105,6 +122,12 @@ export interface InputFieldProps extends BaseProps {
 	paddingInlineStart?: CSSProperties[ 'paddingInlineStart' ];
 	paddingInlineEnd?: CSSProperties[ 'paddingInlineEnd' ];
 	setIsFocused: ( isFocused: boolean ) => void;
+	/**
+	 * Whether to wrap input field in a Tooltip component.
+	 *
+	 * @default false
+	 */
+	showTooltip?: boolean;
```

Comment:
Would allowing custom tooltip text be beneficial or should we be ensuring that any tooltip matches the label?

Reply:
I think we could benefit from allowing consumers of `InputControl` to set a custom tooltip text. We could fallback to `label` in case it's not specified?

Reply:
Added option to set custom tooltip text in https://github.com/WordPress/gutenberg/pull/42726/commits/a858d7092a7bd9ddb73f9c3e8f67547a9c2ce7ff.